packages/arb-token-bridge-ui/src/components/syncers/RetryableTxnsIncluder.tsx

```diff
@@ -79,9 +69,14 @@ const RetryableTxnsIncluder = (): JSX.Element => {
     }
 
     const successfulL1Deposits = actions.app.getSuccessfulL1Deposits()
+    const sortedTransactions = actions.app.getSortedTransactions()
```

Comment:
is it an issue that we're getting this inside an useCallback but not adding this to the input array in [here](https://github.com/OffchainLabs/arb-token-bridge/blob/0c72f10779f5656c6cabf3ca6efb3ae9428dbbc2/packages/arb-token-bridge-ui/src/components/syncers/RetryableTxnsIncluder.tsx#L139)?

i'm not sure how the store deals with the memoization

Reply:
overmind does the work of keeping the derived state up to date, should be chill